tools/espeasyupdate

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+PROG=$( basename $0 .sh )
+
+[[ "$1" == "-f" ]] && FORCEUPDATE=1 && shift
+[[ "$1" == "-n" ]] && NOUPDATE=1 && shift
+[[ "$1" == "-f" ]] && FORCEUPDATE=1 && shift
```

Comment:
Parsing arguments this way isn't very efficient, and also has a fixed argument order, why not use `getopts` ?

Reply:
Getopts does not add Munch value In this case. The 3 lines is just so that any order Will do